### DISCUSSION TITLE
Tune timings

### DISCUSSION THREAD

**Discussion by odflor** [frntc/RAD#25] [General]
Hi!

How I can do to "tune" the timings of the RAD?

How I can know what timings is better to my c128 NTSC?

What is better, up of down the timing?

Best Regards

**Comment by frntc** [comment]
Either in the .cfg or in the main menu when you press the pound-sign-key.

It's not about larger or smaller values, but typically you only need to touch the last three values:
- RW BEFORE ADDR: default is 40, some boards need a value between 57 and 63 (rough estimate!)
- ENABLE RW+ADDR and ENABLE DATA: the need to be a bit apart (~150 to 200), but the former can work at values such as 470/680, but also at 600/800. Seems to be something with the RAM, but still needs to be figured out.
